firestore/app/src/main/java/com/google/firebase/example/fireeats/kotlin/MainFragment.kt

```diff
@@ -53,15 +53,15 @@ class MainFragment : Fragment(),
     private lateinit var viewModel: MainActivityViewModel
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-        binding = FragmentMainBinding.inflate(inflater, container, false);
-        return binding.root;
+        binding = FragmentMainBinding.inflate(inflater, container, false)
+        return binding.root
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
         // View model
-        viewModel = ViewModelProvider(this).get(MainActivityViewModel::class.java)
+        viewModel = ViewModelProvider(this)[MainActivityViewModel::class.java]
```

Comment:
I think there's [an extension function](https://developer.android.com/reference/androidx/lifecycle/ViewModelProvider#(androidx.lifecycle.ViewModelProvider).get()) that allows us to use `get<VM>()` instead? I would prefer that because it's easier to read rather that this.

```suggestion
        viewModel = ViewModelProvider(this).get<MainActivityViewModel>()
```

Reply:
instead finish!